Monika After Story/game/definitions.rpy

```diff
@@ -5500,6 +5500,19 @@ init -1 python:
 
         RETURNS: True if a proccess in proc_list is running, False otherwise
         """
+        if renpy.linux:
+            for proc in proc_list:
+                import subprocess
```

Comment:
At this point, the subprocess module is already imported globally. We don't need to import it again.
```suggestion
```

Reply:
Ah, true. Not sure how this slipped in.